### ISSUE TITLE
ld: library not found for -lVasern

### ISSUE THREAD

**Issue by kailashvele** [vasern/vasern#39]
ld: library not found for -lVasern
clang: error: linker command failed with exit code 1 (use -v to see invocation)

Getting this error on Archiving project in XCode.

Since I am using cocoapods for Firebase I have to open .xcworkspace file in XCode.

However, I have linked the library in the Projects by both the Auto and Manual methods and followed the standard process of adding libVasern.a in linkers.

The library seems to be linked since running application on an emulator doesn't throw the error while Archiving it does.

Any suggestions?

**Comment by hieunc229** [comment]
Hi Kai, seem like the library isn't linked for production build. Similar to this issue https://github.com/vasern/vasern/issues/27#issuecomment-478375332

Let me know if the issue still persist

**Comment by kailashvele** [comment]
So it turns out, changing the build target from 9 or below to 11 doesn't give this error.

Following steps I followed after connecting with @hieunc229 on Twitter:
1. Change the build target in the Build Settings to 11.
![image](https://user-images.githubusercontent.com/7221983/65215557-8a004300-dacb-11e9-922c-c131640757df.png)

2. Ran build clean.
![image](https://user-images.githubusercontent.com/7221983/65215595-a69c7b00-dacb-11e9-997b-0ea5058236ef.png)

3. And then did the Archive.

Also, note if you have the correct path in header search path ie. $(SRCROOT)/../node_modules/vasern/ios
